coriolis/api/v1/migrations.py

```diff
@@ -106,11 +109,18 @@ def create(self, req, body):
                 raise exc.HTTPBadRequest(
                     explanation="Invalid destination "
                                 "environment: %s" % message)
+            is_valid, message = (
```

Comment:
Please move this check in `_validate_migration_input` to be consistent with the dest-env validation (that one has to be there due to the `network_map` and `storage_mappings` situation with the plugins)